ls1intum/Artemis
# `Communication`: Display improved edited hints

### Checklist
#### General


#### Server
#### Client

### Motivation and Context


Currently, the server simply appends (last edited by xxx on date yyy) to the content of edited messages. We want to improve this.

### Description

1. Added new column updated_date to posting to store if the posting has ever been updated and when so.
2. The client displays (edited) on posts that have been edited.

### Steps for Testing
1. Go to conversations and send a new message
2. Edit the message previously sent
3. You should see (edited) on the post

### Review Progress






#### Code Review
#### Manual Tests

### Test Coverage







### Screenshots


![image](https://github.com/ls1intum/Artemis/assets/100865202/66f268fd-d0f4-46df-b3a7-f9741b4ef63a)
